x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/AutoFollowCoordinator.java

```diff
@@ -327,7 +327,7 @@ private void followLeaderIndex(String autoFollowPattenName,
             followRequest.setMaxConcurrentReadBatches(pattern.getMaxConcurrentReadBatches());
             followRequest.setMaxBatchSize(pattern.getMaxBatchSize());
             followRequest.setMaxConcurrentWriteBatches(pattern.getMaxConcurrentWriteBatches());
-            followRequest.setMaxWriteBufferSize(pattern.getMaxWriteBufferSize());
+            followRequest.setMaxWriteBufferCount(pattern.getMaxWriteBufferCount());
```

Comment:
We forget passing "maxWriteBufferSize" parameter.

Reply:
whoops